src/localization/locales/en-US.json

```diff
@@ -465,7 +502,8 @@
         "validation_field_required": "$t({{path}}) is required",
         "validation_field_required_array": "$t({{path}}) {{arrayIndex}} is required",
         "validation_field_max_length": "{{path}} must be {{max}} characters maximum.",
-        "validation_array_min": "$t({{path}}) must have at least {{min}} item."
+        "validation_array_min": "$t({{path}}) must have at least {{min}} item.",
```

Comment:
issue: if {{min}} > 1, should be items. Use the i18n plural support to allow for both cases.